config/application.rb

```diff
@@ -136,6 +137,12 @@ class Application < Rails::Application
     # Add apidoc hash in headers for smarter caching
     config.middleware.use "Apipie::Middleware::ChecksumInHeaders"
 
+    Foreman::Logging.configure(:log_directory => "#{Rails.root}/log", :environment => Rails.env)
+    config.colorize_logging = Foreman::Logging.config[:colorize]
```

Comment:
Our module doesn't cover logging today, so we can do that at a later stage.  It should work correctly by default without any use of the installer.